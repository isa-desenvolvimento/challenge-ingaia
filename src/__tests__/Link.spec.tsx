import { render } from '@testing-library/react';
import Link from '../components/Link';

describe('<Link />', () => {
  it('should render component Link children', () => {
    const { getByRole } = render(
      <Link href="/">
        <h1>Test Link</h1>
      </Link>,
    );
    expect(getByRole('heading', { name: /test Link/i })).toBeInTheDocument();
  });

  it('should render snapshot', () => {
    const { container } = render(
      <Link href="/">
        <h1>Test Link</h1>
      </Link>,
    );
    expect(container).toMatchSnapshot();
  });
});
