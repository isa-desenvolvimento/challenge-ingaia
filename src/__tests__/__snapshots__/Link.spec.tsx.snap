// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> should render snapshot 1`] = `
<div>
  <a
    href="/"
  >
    <h1>
      Test Link
    </h1>
  </a>
</div>
`;
